std/range/primitives.d

```diff
@@ -126,13 +126,19 @@ auto h = r.front; // can get the front of the range of non-void type
 The semantics of an input range (not checkable during compilation) are
 assumed to be the following ($(D r) is an object of type $(D R)):
 
-$(UL $(LI $(D r.empty) returns $(D false) iff there is more data
-available in the range.)  $(LI $(D r.front) returns the current
-element in the range. It may return by value or by reference. Calling
-$(D r.front) is allowed only if calling $(D r.empty) has, or would
-have, returned $(D false).) $(LI $(D r.popFront) advances to the next
-element in the range. Calling $(D r.popFront) is allowed only if
-calling $(D r.empty) has, or would have, returned $(D false).))
+$(UL
+    $(LI `r.empty` returns `false` iff there is more data
+    available in the range.)
+    $(LI `r.front` returns the current element in the range.
+    It may return by value or by reference. Calling `r.front`
+    is allowed only if calling `r.empty` has, or would have,
+    returned `false`.)
+    $(LI Calling `r.front` multiple times without calling
+    `r.popFront` yields the same result for each call.)
```

Comment:
just a formatting nitpick: it seems that before every method/property has had it's own point. That seems reasonable, Can we keep this?